public/manifest.json

```diff
@@ -0,0 +1,9 @@
+{
```

Comment:
Not sure what is the purpose of this file, but I believe a non-default value should be here.

Reply:
It's a Web App Manifest. Useful when the app is used on mobile devices( name, app icon etc. )